### ISSUE TITLE
Where do the viewing conditions for sRGB in the WCAG techniques page come from?

### ISSUE THREAD

**Issue by svgeesus** [w3c/wcag3#229]
(I hope this is the right repo for this comment. If not, please direct me to the correct one).

This is a request for clarification.

The [Visual contrast of text](https://www.w3.org/WAI/GL/WCAG3/2022/how-tos/visual-contrast-of-text/#develop-button) page states, on the develop tab:

> The designer's workspace is important. Variations in monitor calibration and ambient lighting will affect the final output, often in unexpected ways.

>    Use a hardware calibrator such as an XRite Display, and use software to calibrate the monitor to the sRGB standard.
>    The ambient room lighting should be about 20% of the brightness of the monitor at peak white. For a monitor with a peak white of 160 cd/m2, the ambient should be around 64 lux.
>    Put another way, setting the full screen to sRGB mid grey color #808080, the monitor should appear around the same brightness as the "overall" room lighting.
>    The color temperature of the room lighting should be 5500K, ideally with a CRI over 90.
>    Don't paint the walls 18% grey. Instead, paint the walls pure white and set the lighting level so the white walls approximate the monitor at #808080 grey.
>    And finally, avoid bright colors in the field of vision when looking at a monitor. Nearby colors can affect how you perceive the monitor content.

I certainly approve of specifying the viewing environment for testing, but would like to understand where these guidelines come from, and why they are somewhat different to the sRGB viewing conditions.

The first point, to use a hardware calibrator, is a good one although what exactly should be calibrated (whitepoint, primary chromaticities, Tone Response Curve (EOTF), surround, and ambient illuminance) could use a little more detail.In particular I assume the monitor should cover 100% or more of the sRGB gamut by volume, so that it can accurately be set to emulate sRGB.

The second point sets the ambient illuminance correctly to 64 lux, but tying this to a percentage of media white seems odd (is that where the 160 cd/m<sup>2</sup> comes from? The sRGB specification sets the media white to 80 cd/m<sup>2</sup>, although that value is rarely adhered to in practice with the advent of brighter LCD screens.

Where does the room lighting color temperature of 5500K come from, and is this intended to be a daylight simulator (D55) or some other type? Is the user expected to be adapted to this white, or to the monitor D65 white? (sRGB assumes full adaptation to monitor white).

For reference, the [viewing conditions for Display P3](https://www.color.org/chardata/rgb/DCIP3.xalter) are part of the official specification](https://www.color.org/chardata/rgb/DisplayP3.xalter) and are identical to[ those for sRGB](https://www.color.org/chardata/rgb/srgb.xalter), whose official definition is sadly behind a paywall.

> Image background (proximal field): 16 cd/m2
> Viewing surround: 4.1 cd/m2
> Ambient illuminance: 64 lux
> Adapted white point luminance: Unspecified
> Adapted white point chromaticity: Unspecified

**Comment by cwadamsoforacle** [comment]
This issue has been transferred from "silver" to "WCAG 3".  This particular content is not part of the current published WCAG 3 draft, but the issues raised are still of importance to the research that needs to occur.
